### PR TITLE
Additional update/edits to CHANGELOG.md for 1.1 

### PR DESCRIPTION
**Pull request by ok-reza**
Consistency pass, creates parity with terms found in community_features.md, fixes errors, provides clearer descriptions where needed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,84 +24,76 @@
...
-- Added `VU Meter` rendering in the sidebar in Song / Arranger / Performance Views.
-- Added ability to save a synth/sample drum back to an instrument preset by holding audition and pressing save.
-- Mod (Gold) Encoders learned to the Mod Matrix can now access the full range of the Mod Matrix / Patch Cable parameters (e.g. from -50 to +50).
+- Added Master Chromatic Transpose of All Scale Mode Instrument Clips from any SONG View (SONG, ARRANGER,
+  ARRANGER AUTOMATION and PERFORMANCE VIEW.) Uses the same shortcut as in a Synth/Midi/CV clip (Press and
+  turn `▼︎▲︎` to transpose all clips by +/- 1 semitone). The number of semitones transposed is customizable (Press `SHIFT`
+  + turn `▼︎▲︎`). After transposing, the display shows the new Root Note (and Scale Name on OLED displays).
+- Added `SIDEBAR CONTROLS` in  `KEYBOARD VIEW` for synths. By default the two sidebar columns pertain to `VELOCITY` (red) and `MOD WHEEL` (blue). Holding a pad sets it momentarily to that value, tapping a pad latches it to that value. The functionality can be changed per column by holding the top pad and turning `SELECT` and include 'CHORDS', `CHORD MEMORY`, and `SCALES`.
```
Comment (sapphire-arches) [inline CHANGELOG.md:53]:
```suggestion
- Added `SIDEBAR CONTROLS` in  `KEYBOARD VIEW` for synths. By default the two sidebar columns pertain to `VELOCITY` (red) and `MOD WHEEL` (blue). Holding a pad sets it momentarily to that value, tapping a pad latches it to that value. The functionality can be changed per column by holding the top pad and turning `▼▲`. Available options are `VELOCITY`, `MOD WHEEL`, 'CHORDS', `CHORD MEMORY`, and `SCALES`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,84 +24,76 @@
...
+- Added a `VU METER` toggle that displays the VU Meter on the sidebar in `SONG`, `ARRANGER`, & `PERFORMANCE VIEW`.
```
Comment (sapphire-arches) [inline CHANGELOG.md:54]:
```suggestion
- Added a `VU METER` toggle that displays the VU Meter on the sidebar in `SONG`, `ARRANGER`, & `PERFORMANCE VIEW`. To activate it, on a song view ensure `AFFECT-ENTIRE` is enabled, then select the `LEVEL/PAN` mod button and press it again to activate the meter.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -25,84 +24,76 @@
...
-- Added a new default setting that controls which playback mode new slices of a kit will get. 
-- Added indicators to the waveform loop lock feature for both 7seg (rightmost `.`) and OLED (lock indicator).
+- Added a new default setting that controls which playback mode new slices of a kit will receive. 
+- Added indicators when `WAVEFORM LOOP LOCK` is enabled. 7SEG displays a rightmost `.` on screen and OLED displays a lock icon.
 - Modified waveform marker rendering to improve clarity.
 - Created a new menu hierarchies document that documents the Deluge menu structure for OLED and 7SEG and can be used as a reference for navigating the various menu's. See: [Menu Hierarchies](https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/menu_hierarchies.md)
-- Added MIDI learning of Song Params
-- Added Synth/MIDI/CV clip configuration of note row play direction. Hold audition pad while entering the play direction menu to set the play direction for the selected note row. While in the note row play direction menu, you can select other note rows to quickly set the play directiom for multiple note rows.
+- Added MIDI learning of Song Params.
```
Comment (sapphire-arches) [inline CHANGELOG.md:72]:
```suggestion
```
This has been removed for now.

File: CHANGELOG.md [hunk 4]
```diff
@@ -25,84 +24,76 @@
...
+- Added Synth/MIDI/CV clip configuration of note row play direction. Hold `AUDITION` while entering the `PLAY DIRECTION` menu to set the play direction for the selected note row. While in the note row play direction menu, you can select other note rows to quickly set the play directiom for multiple note rows.
 
 In addition, a number of improvements have been made to how the OLED display is used:
 
-- Added parameter name (including mod matrix / patch cable mappings) to Mod (Gold) Encoder popups.
+- Added parameter names (including mod matrix / patch cable mappings) to Mod (Gold) Encoder popups.
 - `ARRANGER VIEW` and `SONG VIEW` now display the name of the current view on the screen.
 - The 12TET note name is now displayed along with the MIDI note number.
-- Added a new community setting which allows emulating the 7SEG style on the OLED display.
+- Added a new community setting which allows emulating the 7SEG style on the OLED display. When set to 'TOGGLE' press `SHIFT`+`LEARN/INPUT`+`AFFECT ENTIRE` to switch to the emulated 7SEG display.
```
Comment (sapphire-arches) [inline CHANGELOG.md:80]:
```suggestion
- Added a new community setting which allows emulating the 7SEG style on the OLED display. When set to `TOGGLE` press `SHIFT`+`LEARN/INPUT`+`AFFECT ENTIRE` to switch to the emulated 7SEG display. 
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -25,84 +24,76 @@
...
 
-- When changing instrument presets in session or arranger view all presets are now changed. Individual clips can still
-  have their presets changed by first entering the clip.
+- When changing instrument presets in `SONG VIEW` or `ARRANGER VIEW`, all pertaining clips of that instrument switch as well. Individual clips can still have only their preset changed by doing so in `CLIP VIEW`.
 - Added support for 5 and 6 note scales.
 - Added 8 new built-in scales: Melodic Minor, Hungarian Minor, Marva (Indian), Arabian, Whole Tone, Blues, Pentatonic
   Minor, Hirajoshi.
-- Added "play `ONCE`" clip launch mode, settable in `ROW VIEW` (Song view) by pressing the Select Encoder while holding
-  the mute pad of a row. Causes the clip to play once when triggered and then mute without changing the section.
+- Added `ONCE CLIP` launch mode, which allows a clip to play just once when launched and then mute itself. Settable in Song's `ROWS VIEW` by holding the `MUTE` pad of a row and then pressing `SELECT`. Causes
```
Comment (sapphire-arches) [inline CHANGELOG.md:90]:
```suggestion
- Added `ONCE CLIP` launch mode, which allows a clip to play just once when launched and then mute itself. Settable in Song's `ROWS VIEW` by holding the `MUTE` pad of a row and then pressing `SELECT`. 
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -25,84 +24,76 @@
...
-  and turn `▼︎▲︎`). After transposing the display show the new Root Note (and Scale Name if you have an OLED display).
-- Added sidebar functionality in keyboard screens - by default it is velocity (red) and mod wheel (blue), holding a pad
-  sets it momentarily and tapping latches it. The functionality can be changed by holding the top pad and scrolling
-  select.
-- Updated the count-in setting to allow specifying the number of bars (1-4).
-- Added `VU Meter` rendering in the sidebar in Song / Arranger / Performance Views.
-- Added ability to save a synth/sample drum back to an instrument preset by holding audition and pressing save.
-- Mod (Gold) Encoders learned to the Mod Matrix can now access the full range of the Mod Matrix / Patch Cable parameters (e.g. from -50 to +50).
+- Added Master Chromatic Transpose of All Scale Mode Instrument Clips from any SONG View (SONG, ARRANGER,
+  ARRANGER AUTOMATION and PERFORMANCE VIEW.) Uses the same shortcut as in a Synth/Midi/CV clip (Press and
```
[the remark this answers is not in the document]
Comment (seangoodvibes) [reply CHANGELOG.md:50]: Change `Arranger Automation` to `Arranger Automation View` please! thx

**Review by seangoodvibes — APPROVED** [review]

**Pull request merged**